### PR TITLE
[ES] Add `Notelock`

### PR DESCRIPTION
**Pull request by Ekkusu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - notas bloqueadas
+  - notas temblorosas
+  - agitando notas
+---
+
+# Notelock
+
+![](img/notelock.gif "Un ejemplo del Notelock: la segunda nota es inaccesible")
+
+El **Notelock**, o **nota de bloqueo**, es un término informal para la mecánica del juego [osu!](/wiki/Game_mode/osu!), el cual puede evitar que un jugador elimine un objeto golpeado. Esto pasa si **dos** condiciones se cumplen al mismo tiempo:
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:15]:
```suggestion
El **Notelock**, o **nota de bloqueo**, es un término informal para la mecánica del juego [osu!](/wiki/Game_mode/osu!), el cual puede evitar que un jugador elimine un objeto de golpeoo. Esto pasa si **dos** condiciones se cumplen al mismo tiempo:
```

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - agitando notas
+---
+
+# Notelock
+
+![](img/notelock.gif "Un ejemplo del Notelock: la segunda nota es inaccesible")
+
+El **Notelock**, o **nota de bloqueo**, es un término informal para la mecánica del juego [osu!](/wiki/Game_mode/osu!), el cual puede evitar que un jugador elimine un objeto golpeado. Esto pasa si **dos** condiciones se cumplen al mismo tiempo: 
+
+1. La [ventana de tiempo](/wiki/Beatmapping/Overall_difficulty#timing) de dos objetos de golpeo se superpone.
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:17]:
```suggestion
1. Las [ventanas de tiempo](/wiki/Beatmapping/Overall_difficulty#timing) de dos objetos de golpeo se superponen.
```

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+2. El primer objeto de los dos aún no ha sido juzgado (acertado o fallado).
+
+En este caso, se dice que el segundo objeto está *bloqueado* ante el primero, lo que hace que osu! ignore los golpeos del jugador hasta que haya pasado la ventana de tiempo del primer objeto. El Notelock solo es un problema si el jugador no puede volver al objeto anterior e interactuar con el correctamente, así se quita el bloqueo. En ese caso, una nota bloqueada puede causar un fallo en cascada, forzando al jugador a fallar más y más objetos posteriores, hasta quedar sin [salud](/wiki/Gameplay/Health) y falle el mapa.
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:20]:
```suggestion
En este caso, se dice que el segundo objeto está *bloqueado* ante el primero, lo que hace que osu! ignore los golpes del jugador hasta que haya pasado la ventana de golpeo del primer objeto. El Notelock solo es un problema si el jugador no puede volver al objeto anterior e interactuar con el correctamente, así se quita el bloqueo. En ese caso, una nota bloqueada puede causar un fallo en cascada, forzando al jugador a fallar más y más objetos posteriores, hasta quedar sin [salud](/wiki/Gameplay/Health) y fallar el mapa.
```

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Cuando ocurra el Notelock, el circulo golpeado se sacudirá. Esto no sucede con deslizadores o ruletas.
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:22]:
```suggestion
Cuando se produce el notelock, el circulo golpeado se sacudirá. Esto no sucede con deslizadores o ruletas.
```

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Causa
+
+El notelock es parte del sistema de la ventana de tempo de osu!, y sucede cuando la ventana de tiempo de dos objetos se superpone. Esto ocurre con mayor frecuencia en mapas con valores bajos de [OD](/wiki/Beatmapping/Overall_difficulty) o valores altos de [BPM](/wiki/Beatmapping/Beats_per_minute), ya que en estos casos son más frecuentes la superposición en la ventana de tiempo.
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:26]:
```suggestion
El notelock es parte del sistema de la ventana de tempo de osu!, y sucede cuando la ventana de tiempo de dos objetos se superponen. Esto ocurre con mayor frecuencia en mapas con valores bajos de [OD](/wiki/Beatmapping/Overall_difficulty) o valores altos de [BPM](/wiki/Beatmapping/Beats_per_minute), ya que en estos casos son más frecuentes la superposición en la ventana de tiempo.
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Como se Juega/Juicio/Bloqueo de notas/es.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+tags:
```
Comment (agatemosu) [inline wiki/Como se Juega/Juicio/Bloqueo de notas/es.md:2]:
```suggestion
tags:
```

**Comment by agatemosu** [comment]
You don't have to translate the file path

**Comment by Ekkusu** [comment]
> You don't have to translate the file path

Not again T_T, that's a bug I don't know how to fix. Do you happen to know how to do it so you don't have to redo the code with a new push request (and close this one)?

File: wiki/Como se Juega/Juicio/Bloqueo de notas/es.md (hunk 6 above)
Comment (Ekkusu) [reply wiki/Como se Juega/Juicio/Bloqueo de notas/es.md:2]: Oh, esto es muy útil, muchas gracias

**Comment by Walavouchey** [comment]
did it for you

have a look at [this part of the contribution guide](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews), you can apply reviews in batches automatically from the "files changed" tab, without having to edit the file manually

**Comment by Ekkusu** [comment]
> did it for you
> 
> have a look at [this part of the contribution guide](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews), you can apply reviews in batches automatically from the "files changed" tab, without having to edit the file manually

Thank you very much, I'll make sure to use it next time 👍  (and prevent the file location from being changed again)

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+# Notelock
+
+![](img/notelock.gif "Un ejemplo del Notelock: la segunda nota es inaccesible")
+
+El **Notelock**, o **nota de bloqueo**, es un término informal para la mecánica del juego [osu!](/wiki/Game_mode/osu!), el cual puede evitar que un jugador elimine un objeto de golpeo. Esto pasa si **dos** condiciones se cumplen al mismo tiempo:
+
+1. Las [ventanas de tiempo](/wiki/Beatmapping/Overall_difficulty#timing) de dos objetos de golpeo se superponen.
+2. El primer objeto de los dos aún no ha sido juzgado (acertado o fallado).
+
+En este caso, se dice que el segundo objeto está *bloqueado* ante el primero, lo que hace que osu! ignore los golpes del jugador hasta que haya pasado la ventana de golpeo del primer objeto. El Notelock solo es un problema si el jugador no puede volver al objeto anterior e interactuar con el correctamente, así se quita el bloqueo. En ese caso, una nota bloqueada puede causar un fallo en cascada, forzando al jugador a fallar más y más objetos posteriores, hasta quedar sin [salud](/wiki/Gameplay/Health) y fallar el mapa.
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:20]:
```suggestion
En este caso, se dice que el segundo objeto está *bloqueado* ante el primero, lo que hace que osu! ignore los golpes del jugador hasta que haya pasado la ventana de golpeo del primer objeto. El Notelock solo es un problema si el jugador no puede volver al objeto anterior e interactuar con él correctamente, así se quita el bloqueo. En ese caso, una nota bloqueada puede causar un fallo en cascada, forzando al jugador a fallar más y más objetos posteriores, hasta quedar sin [salud](/wiki/Gameplay/Health) y fallar el mapa.
```

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+Cuando se produce el notelock, el circulo golpeado se sacudirá. Esto no sucede con deslizadores o ruletas.
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:22]:
```suggestion
Cuando se produce el notelock, el círculo de golpeo se sacudirá. Esto no sucede con deslizadores o ruletas.
```

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+## Causa
+
+El notelock es parte del sistema de la ventana de tempo de osu!, y sucede cuando la ventana de tiempo de dos objetos se superponen. Esto ocurre con mayor frecuencia en mapas con valores bajos de [OD](/wiki/Beatmapping/Overall_difficulty) o valores altos de [BPM](/wiki/Beatmapping/Beats_per_minute), ya que en estos casos son más frecuentes la superposición en la ventana de tiempo.
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:26]:
```suggestion
El Notelock es parte del sistema de la ventana de tempo de osu!, y sucede cuando la ventana de tiempo de dos objetos se superponen. Esto ocurre con mayor frecuencia en mapas con valores bajos de [OD](/wiki/Beatmapping/Overall_difficulty) o valores altos de [BPM](/wiki/Beatmapping/Beats_per_minute), ya que en estos casos son más frecuentes la superposición en la ventana de tiempo.
```

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+Dado que regularmente los objetos en los mapas de osu! están destinados a borrarse en orden cronológico, el rechazo de golpes causados por el notelock, generalmente juega un papel positivos como:
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Notelock/es.md:28]:
```suggestion
Dado que regularmente los objetos en los mapas de osu! están destinados a borrarse en orden cronológico, el rechazo de golpes causados por el notelock, generalmente juega un papel positivo como:
```

**Review by agatemosu — DISMISSED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**